src/driver.rs

```diff
@@ -168,7 +168,7 @@ impl Driver {
     /// ds.set_spatial_ref(&SpatialRef::from_epsg(4326)?)?;
     /// assert_eq!(ds.raster_count(), 1);
     /// assert_eq!(ds.raster_size(), (64, 64));
-    /// assert_eq!(ds.rasterband(1)?.band_type(), u8::gdal_ordinal());
+    /// assert_eq!(ds.rasterband(1)?.band_type(), GdalDataType::UInt8);
```

Comment:
This could also be written `u8::datatype()` given the correct imports, but using the enum directly is more intuitive, IMO.